libraries/core_libs/node/src/node.cpp

```diff
@@ -84,23 +84,29 @@ void FullNode::init() {
   }
   LOG(log_nf_) << "DB initialized ...";
 
+  std::cout << "conf_.chain.genesisHash() 2: " << conf_.chain.genesisHash() << std::endl;
+  std::cout << enc_json(conf_.chain.final_chain.state) << std::endl;
```

Comment:
std cout ?

Reply:
I think I mentioned yesterday that I have some issues with tests, so this is here because of that. 
P.S. it is still a draft :)